python/ray/train/tests/test_backend.py

```diff
@@ -547,6 +548,76 @@ def get_resources():
     assert results == expected_results
 
 
+@pytest.mark.parametrize(
+    "worker_results",
+    [
+        (1, [[0]]),
+        (2, [[0, 1]] * 2),
+        (3, [[0]] + [[0, 1]] * 2),
+        (4, [[0, 1]] * 4),
+    ],
+)
+def test_npu_accelerator_ids(ray_2_node_2_npus, worker_results):
```

Comment:
i think we don't need to test every single accelerator for this device id sharing behavior. I'd like to keep the tests as accelerator-agnostic as possible.

Reply:
Sure, removed this test.